src/components/skip-link/_skip-link.scss

```diff
@@ -11,5 +11,13 @@
 
     display: block;
     padding: govuk-spacing(2) govuk-spacing(3);
+
+    // Add safe area (e.g. iPhone X, Google Pixel 3)
+    @supports (padding: unquote("max(calc(0px))")) {
+      // Use safe area or default padding, whichever is larger
+      // Escaped due to Sass max() vs. CSS native max()
+      padding-right: unquote("max(#{govuk-spacing(3)}, calc(#{$govuk-gutter-half} + env(safe-area-inset-right)))");
```

Comment:
Should we be using `govuk-spacing(3)` within the calc as well?

Reply:
Anything that ends in "px" triggers a match here, happy to if you prefer?

Reply:
Non-blocking, but it might make it more obvious where these values come from if we set some private variables that can be shared between the first definition and the notch definitions?

Reply:
Just to be clear, I'm asking if it's intentional that we're using _both_ `$govuk-spacing(3)` and `$govuk-gutter-half` on this line – on the two different sides of the `max` calculation.

Reply:
I believe the logic will go something like this:

Does the device have an inset?

If yes, then we want that inset + a reduced gutter, since a regular gutter size is optically too large.

If there's no inset, then use the regular gutter.

Reply:
I'm using `$govuk-spacing(3)` only for **_skip-link.scss**
I'm using `$govuk-gutter[-half]` only for **_width-container.scss**

Cheers for spotting where I'd used both in places.

This diff is outdated, so add a new comment if you like?